ksp_plugin_adapter/flight_planner.cs

```diff
@@ -406,6 +403,14 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
     }
   }
 
+  internal void ModifyIthBurn(string vessel_guid, int i, Burn burn) {
+    var status = plugin.FlightPlanReplace(vessel_guid, burn, i);
+    UpdateStatus(status, i);
+    if (burn_editors_?.Count > 0) {
```

Comment:
To be defensive, this test should probably check that there is actually a burn editor at index `i`.